hack/kind-config.yaml

```diff
@@ -0,0 +1,12 @@
+# three node (two workers) cluster config for github workflow!
```

Comment:
NIT:
Is it three nodes where two of them are only workers and another node is both worker and control-plane or it is three worker nodes and additional node which is control-plane?

Reply:
good catch, I started with a 1 cp + 2 w cluster, but that didn't work with the default minHEalthy of 51%. So I increased worker count and forgot to update the comment. Fixed